hooks/post-command

```diff
@@ -112,6 +112,11 @@ if [[ "${BUILDKITE_PLUGIN_TRIVY_IGNORE_UNFIXED:-false}" == true ]] ; then
   echo "ignore-unfixed is set. Will ignore unfixed vulnerabilities"
 fi
 
+if [[ -n "${BUILDKITE_PLUGIN_TRIVY_KUBE_VERSION:-}" ]] ; then
+  fsargs+=("--helm-kube-version" "${BUILDKITE_PLUGIN_TRIVY_KUBE_VERSION}")
+  echo "using non-default kube-version"
+fi
+
```

Comment:
looks like trivy accepts `helm-kube-version` flag regardless of which subcommand or scanner is used, so we should be ok